apps/ideaspace/src/components/common/IdeaCard/IdeaCardUpdated.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { atoms } from '@devlaunchers/components/src/components';
+import timeSvg from '../../../images/time.svg';
+
+const IdeaCardUpdated = ({
+    updatedAt
+}) => {
+    const [updated, setUpdated] = useState('');
+
+    React.useEffect(() => {
+        const toBeOneSecond = 1000;
+        const secondsInDay = 86400;
+        const secondsInMonth = secondsInDay * 30;
+        const secondsInYear = secondsInMonth * 12;
+        const secondsInHour = 3600;
+        const secondsInMinute = 60;
+
+        const difftime = (new Date() - new Date(updatedAt)) / toBeOneSecond;
+
+        if (difftime > secondsInYear) {
```

Comment:
Just a thought, and not necessary to change as part of this ticket-it may help to put this method in a helper file so it could be reused in other components. For example this would be really helpful to reuse for comments and some other places in the app.